Packs/ThreatGrid/Playbooks/playbook-Detonate_File_-_ThreatGrid.yml

```diff
@@ -1,24 +1,24 @@
-id: detonate_file_-_threatgrid
+id: 06f52f43-59cb-4a1e-8492-5ecd612302a7
 version: -1
 name: Detonate File - ThreatGrid
-description: Detonate one or more files using the ThreatGrid integration. This playbook returns relevant reports to the War Room and file reputations to the context data. The detonation supports the following file types - EXE, DLL, JAR, JS, PDF, DOC, DOCX, RTF, XLS, PPT, PPTX, XML, ZIP, VBN, SEP, XZ, GZ, BZ2, TAR, MHTML, SWF, LNK, URL, MSI, JTD, JTT, JTDC, JTTC, HWP, HWT, HWPX, BAT, HTA, PS1, VBS, WSF, JSE, VBE, CHM
```

Comment:
why the change?

Reply:
We updated this playbook so the description was changed. I will update this description again.